invenio_records_lom/fixtures/demo.py

```diff
@@ -0,0 +1,363 @@
+        "description": langstringify(fake, fake.paragraph()),
+        "language": [create_fake_language(fake) for __ in range(2)],
+    }
+
+
+def create_fake_rights(fake: Faker) -> dict:
+    """Create a fake "rights"-element, compatible with LOMv1.0-standard."""
+    return {
+        "cost": vocabularify(fake, ["yes", "no"]),
+        "copyrightAndOtherRestrictions": vocabularify(fake, ["yes", "no"]),
+        "description": langstringify(fake, fake.paragraph()),
+    }
+
+
+def create_fake_relation(fake: Faker) -> dict:
+    """Create a fake "relation"-element, compatible with LOMv1.0-standard."""
+    kinds = [
+        "ispartof",
+        "haspart",
+        "isversionof",
+        "hasversion",
+        "isformatof",
+        "hasformat",
+        "references",
+        "isreferencedby",
+        "isbasedon",
+        "isbasisfor",
+        "requires",
+        "isrequiredby",
+    ]
+
+    return {
+        "kind": vocabularify(fake, kinds),
+        "resource": {
+            "identifier": [create_fake_identifier(fake) for __ in range(2)],
+            "description": [langstringify(fake, fake.paragraph()) for __ in range(2)],
+        },
+    }
+
+
+def create_fake_annotation(fake: Faker) -> dict:
+    """Create a fake "annotation"-element, compatible with LOMv1.0-standard."""
+    return {
+        "entity": create_fake_vcard(fake),
+        "date": create_fake_datetime(fake),
+        "description": langstringify(fake, fake.paragraph()),
+    }
+
+
+def create_fake_classification(fake: Faker) -> dict:
+    """Create a fake "classification"-element, compatible with LOMv1.0-standard."""
+    purposes = [
+        "discipline",
+        "idea",
+        "prerequisite",
+        "educational objective",
+        "accessability restrictions",
+        "educational level",
+        "skill level",
+        "security level",
+        "competency",
+    ]
+
+    return {
+        "purpose": vocabularify(fake, purposes),
+        "taxonPath": [create_fake_taxonpath(fake) for __ in range(2)],
+        "description": langstringify(fake, fake.paragraph()),
+        "keyword": langstringify(fake, fake.word()),
+    }
+
+
+def create_fake_taxonpath(fake: Faker) -> dict:
+    """Create a fake "taxonPath"-element, compatible with LOMv1.0-standard."""
+    return {
+        "source": langstringify(fake, fake.word()),
+        "taxon": [create_fake_taxon(fake) for __ in range(2)],
+    }
+
+
+def create_fake_taxon(fake: Faker) -> dict:
+    """Create a fake "taxon"-element, compatible with LOMv1.0-standard."""
+    return {
+        "id": fake.lexify(
+            "?????",
+            letters="ABCDEFGHIJKLMNOPQRSTUVWXYZ.0123456789",
+        ),
+        "entry": langstringify(fake, fake.word()),
+    }
+
+
+# ----- functions for creating LOMv1.0-fakes -----
+def create_fake_record(fake: Faker) -> dict:
+    """Create a fake json-representation of a "lom"-element, compatible with LOMv1.0-standard."""
+    data_to_use = {
+        "lom": {
```

Comment:
I am not sure, but i think that "lom" attribute could lead to problems, because in the datamodel there is then the metadata attribute where the metadata dict is stored, and this is all what is in the "lom" dict. so there would then be something like '"metadata": {"lom": {...}}', and i think thats not good.